### PR TITLE
Slides 06 Managing Validation Projects

### PR DESCRIPTION
**Pull request by thebioengineer**
Initial draft - working on adding a waterflow and agile gantt chart example to show that both are supported and you aren't locked down into a specific development process with this framework. its organizational.

### CODE REVIEW

**Comment by thebioengineer** [comment]
@mariev - updated the readme and added triage instructions for attendees and TA's.

File: Debugging_Triage.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+## Debugging
+
+When working through these slides and exercises, don't become discouraged if something doesnt work out or isnt clear. Follow these instructions
```
Comment (mariev) [inline Debugging_Triage.md:3]:
```suggestion
When working through these slides and exercises, don't become discouraged if something does not work immediately or seems obtuse. Follow these instructions:
```

File: Debugging_Triage.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+### Attendee
+
+First, see if any of the other members of the breakout session are coming across similar difficulties. 
+
+Optionally, see if you can reproduce the issue. 
+Try re-starting the assignment and following the instructions. 
+If you still come across errors, reach out to the TA's.
```
Comment (mariev) [inline Debugging_Triage.md:11]:
```suggestion
If you still come across errors, reach out to the TAs.
```

**Review by mariev — APPROVED** [review]
Just some minor contraction usage

**Pull request merged**